scala_proto/scala_proto.bzl

```diff
@@ -323,7 +323,9 @@ def _root_path(f):
     return '/'.join([f.root.path, f.owner.workspace_root])
 
 def _colon_paths(data):
-    return ':'.join(["{root},{path}".format(root=_root_path(f), path=f.path) for f in data])
+    if type(data) != "list":
```

Comment:
Why is this needed? Can we maybe have a test which demonstrates it?

Reply:
Yeah I’d rather have only one input type here: depset or list. Dynamically switching worries me.

Reply:
I mostly wanted to make these things more robust since we have more failures at runtime with no types... but i'll remove it.